hack/update-codecgen.sh

```diff
@@ -117,7 +117,7 @@ fi
 
 echo "Building codecgen"
 CODECGEN="${PWD}/codecgen_binary"
-godep go build -o "${CODECGEN}" github.com/ugorji/go/codec/codecgen
+go build -o "${CODECGEN}" ./vendor/github.com/ugorji/go/codec/codecgen
```

Comment:
leading ./ tells go tools to look for a file relative to CWD.  Without that it looks for $GOPATH/vendor/github.com/ugorji/go/codec/codecgen, which obviously is wrong.